diff --git a/source/bindbc/imgui/bind/imgui.d b/source/bindbc/imgui/bind/imgui.d
index cac9b01..b1fa50e 100644
--- a/source/bindbc/imgui/bind/imgui.d
+++ b/source/bindbc/imgui/bind/imgui.d
@@ -2720,7 +2720,7 @@ version (BindImGui_Static) {
         void igDockContextQueueUndockWindow(ImGuiContext* ctx, ImGuiWindow* window);
         ImGuiComboPreviewData* ImGuiComboPreviewData_ImGuiComboPreviewData();
         void igEndTable();
-        const ImWchar* ImFontAtlas_GetGlyphRangesChineseFull(ImFontAtlas* self);
+        ImWchar* ImFontAtlas_GetGlyphRangesChineseFull(ImFontAtlas* self);
         void igBringWindowToDisplayFront(ImGuiWindow* window);
         ImDrawList* igGetForegroundDrawList_Nil();
         ImDrawList* igGetForegroundDrawList_ViewportPtr(ImGuiViewport* viewport);
@@ -2734,7 +2734,7 @@ version (BindImGui_Static) {
         void igImMax(ImVec2* pOut, const ImVec2 lhs, const ImVec2 rhs);
         void ImRect_GetTR(ImVec2* pOut, ImRect* self);
         void igTableSetupColumn(const(char)* label, ImGuiTableColumnFlags flags = ImGuiTableColumnFlags.None, float init_width_or_weight = 0.0f, ImGuiID user_id = 0);
-        const ImWchar* ImFontAtlas_GetGlyphRangesThai(ImFontAtlas* self);
+        ImWchar* ImFontAtlas_GetGlyphRangesThai(ImFontAtlas* self);
         void ImGuiInputTextState_ClearSelection(ImGuiInputTextState* self);
         void ImFont_GrowIndex(ImFont* self, int new_size);
         void igClosePopupsOverWindow(ImGuiWindow* ref_window, bool restore_focus_to_window_under_popup);
@@ -2772,7 +2772,7 @@ version (BindImGui_Static) {
         void igGetAllocatorFunctions(ImGuiMemAllocFunc* p_alloc_func, ImGuiMemFreeFunc* p_free_func, void** p_user_data);
         void igRenderRectFilledRangeH(ImDrawList* draw_list, const ImRect rect, ImU32 col, float x_start_norm, float x_end_norm, float rounding);
         void igSetWindowDock(ImGuiWindow* window, ImGuiID dock_id, ImGuiCond cond);
-        const ImFontBuilderIO* igImFontAtlasGetBuilderForStbTruetype();
+        ImFontBuilderIO* igImFontAtlasGetBuilderForStbTruetype();
         ImGuiOldColumns* igFindOrCreateColumns(ImGuiWindow* window, ImGuiID id);
         void* ImGuiStorage_GetVoidPtr(ImGuiStorage* self, ImGuiID key);
         int ImGuiInputTextState_GetRedoAvailCount(ImGuiInputTextState* self);
@@ -2919,7 +2919,7 @@ version (BindImGui_Static) {
         void igTabItemLabelAndCloseButton(ImDrawList* draw_list, const ImRect bb, ImGuiTabItemFlags flags, ImVec2 frame_padding, const(char)* label, ImGuiID tab_id, ImGuiID close_button_id, bool is_contents_visible, bool* out_just_closed, bool* out_text_clipped);
         ImGuiTableColumnSortSpecs* ImGuiTableColumnSortSpecs_ImGuiTableColumnSortSpecs();
         void igLogToClipboard(int auto_open_depth = -1);
-        const ImWchar* ImFontAtlas_GetGlyphRangesKorean(ImFontAtlas* self);
+        ImWchar* ImFontAtlas_GetGlyphRangesKorean(ImFontAtlas* self);
         void ImFontGlyphRangesBuilder_SetBit(ImFontGlyphRangesBuilder* self, size_t n);
         void igLogSetNextTextDecoration(const(char)* prefix, const(char)* suffix);
         void igStyleColorsClassic(ImGuiStyle* dst = null);
@@ -2942,7 +2942,7 @@ version (BindImGui_Static) {
         void igEndListBox();
         ImGuiItemStatusFlags igGetItemStatusFlags();
         void igPopFocusScope();
-        const ImVec4* igGetStyleColorVec4(ImGuiCol idx);
+        ImVec4* igGetStyleColorVec4(ImGuiCol idx);
         ImGuiTable* igTableFindByID(ImGuiID id);
         void igShutdown(ImGuiContext* context);
         void igDockBuilderRemoveNodeDockedWindows(ImGuiID node_id, bool clear_settings_refs = true);
@@ -3000,7 +3000,7 @@ version (BindImGui_Static) {
         void igImTriangleBarycentricCoords(const ImVec2 a, const ImVec2 b, const ImVec2 c, const ImVec2 p, float* out_u, float* out_v, float* out_w);
         void ImFontGlyphRangesBuilder_AddRanges(ImFontGlyphRangesBuilder* self, const ImWchar* ranges);
         void igTableSetBgColor(ImGuiTableBgTarget target, ImU32 color, int column_n = -1);
-        const ImWchar* ImFontAtlas_GetGlyphRangesVietnamese(ImFontAtlas* self);
+        ImWchar* ImFontAtlas_GetGlyphRangesVietnamese(ImFontAtlas* self);
         ImGuiContextHook* ImGuiContextHook_ImGuiContextHook();
         const(char)* igGetVersion();
         ImDrawList* ImDrawList_ImDrawList(const ImDrawListSharedData* shared_data);
@@ -3063,7 +3063,7 @@ version (BindImGui_Static) {
         void igLogToBuffer(int auto_open_depth = -1);
         void ImGuiTableTempData_destroy(ImGuiTableTempData* self);
         void* igImFileLoadToMemory(const(char)* filename, const(char)* mode, size_t* out_file_size = null, int padding_bytes = 0);
-        const ImWchar* ImFontAtlas_GetGlyphRangesCyrillic(ImFontAtlas* self);
+        ImWchar* ImFontAtlas_GetGlyphRangesCyrillic(ImFontAtlas* self);
         void ImGuiStyle_destroy(ImGuiStyle* self);
         void ImDrawList_destroy(ImDrawList* self);
         void ImVec4_destroy(ImVec4* self);
@@ -3110,13 +3110,13 @@ version (BindImGui_Static) {
         bool igScrollbarEx(const ImRect bb, ImGuiID id, ImGuiAxis axis, float* p_scroll_v, float avail_v, float contents_v, ImDrawFlags flags);
         void ImDrawList_ChannelsMerge(ImDrawList* self);
         void igSetAllocatorFunctions(ImGuiMemAllocFunc alloc_func, ImGuiMemFreeFunc free_func, void* user_data = null);
-        const ImFontGlyph* ImFont_FindGlyph(ImFont* self, ImWchar c);
+        ImFontGlyph* ImFont_FindGlyph(ImFont* self, ImWchar c);
         void igErrorCheckEndWindowRecover(ImGuiErrorLogCallback log_callback, void* user_data = null);
         int igDockNodeGetDepth(const ImGuiDockNode* node);
         void igDebugStartItemPicker();
         void ImGuiNextWindowData_destroy(ImGuiNextWindowData* self);
         bool ImGuiPayload_IsDelivery(ImGuiPayload* self);
-        const ImWchar* ImFontAtlas_GetGlyphRangesJapanese(ImFontAtlas* self);
+        ImWchar* ImFontAtlas_GetGlyphRangesJapanese(ImFontAtlas* self);
         bool ImRect_Overlaps(ImRect* self, const ImRect r);
         void igCaptureMouseFromApp(bool want_capture_mouse_value = true);
         ImGuiID igAddContextHook(ImGuiContext* context, const ImGuiContextHook* hook);
@@ -3285,7 +3285,7 @@ version (BindImGui_Static) {
         void igSetNextWindowSizeConstraints(const ImVec2 size_min, const ImVec2 size_max, ImGuiSizeCallback custom_callback = null, void* custom_callback_data = null);
         void igTableGcCompactTransientBuffers_TablePtr(ImGuiTable* table);
         void igTableGcCompactTransientBuffers_TableTempDataPtr(ImGuiTableTempData* table);
-        const ImFontGlyph* ImFont_FindGlyphNoFallback(ImFont* self, ImWchar c);
+        ImFontGlyph* ImFont_FindGlyphNoFallback(ImFont* self, ImWchar c);
         bool igShowStyleSelector(const(char)* label);
         void igNavMoveRequestForward(ImGuiDir move_dir, ImGuiDir clip_dir, ImGuiNavMoveFlags move_flags);
         void igNavInitWindow(ImGuiWindow* window, bool force_reinit);
@@ -3313,7 +3313,7 @@ version (BindImGui_Static) {
         void igSetScrollFromPosY_Float(float local_y, float center_y_ratio = 0.5f);
         void igSetScrollFromPosY_WindowPtr(ImGuiWindow* window, float local_y, float center_y_ratio);
         bool igColorButton(const(char)* desc_id, const ImVec4 col, ImGuiColorEditFlags flags = ImGuiColorEditFlags.None, ImVec2 size = ImVec2(0,0));
-        const ImGuiPayload* igAcceptDragDropPayload(const(char)* type, ImGuiDragDropFlags flags = ImGuiDragDropFlags.None);
+        ImGuiPayload* igAcceptDragDropPayload(const(char)* type, ImGuiDragDropFlags flags = ImGuiDragDropFlags.None);
         void igDockContextShutdown(ImGuiContext* ctx);
         void ImDrawList_PopClipRect(ImDrawList* self);
         float igGetCursorPosX();
@@ -3366,7 +3366,7 @@ version (BindImGui_Static) {
         void igLoadIniSettingsFromMemory(const(char)* ini_data, size_t ini_size = 0);
         void igIndent(float indent_w = 0.0f);
         void igPopStyleVar(int count = 1);
-        const ImGuiPlatformMonitor* igGetViewportPlatformMonitor(ImGuiViewport* viewport);
+        ImGuiPlatformMonitor* igGetViewportPlatformMonitor(ImGuiViewport* viewport);
         void igSetNextWindowSize(const ImVec2 size, ImGuiCond cond = ImGuiCond.None);
         bool igInputFloat3(const(char)* label, float[3]*/*[3]*/ v, const(char)* format = "%.3f", ImGuiInputTextFlags flags = ImGuiInputTextFlags.None);
         bool igIsKeyDown(int user_key_index);
@@ -3508,7 +3508,7 @@ version (BindImGui_Static) {
         void ImGuiIO_destroy(ImGuiIO* self);
         void igEndDragDropSource();
         void ImGuiStackSizes_SetToCurrentState(ImGuiStackSizes* self);
-        const ImGuiPayload* igGetDragDropPayload();
+        ImGuiPayload* igGetDragDropPayload();
         void igGetPopupAllowedExtentRect(ImRect* pOut, ImGuiWindow* window);
         void ImGuiStorage_SetInt(ImGuiStorage* self, ImGuiID key, int val);
         void ImGuiWindow_MenuBarRect(ImRect* pOut, ImGuiWindow* self);
@@ -3560,7 +3560,7 @@ version (BindImGui_Static) {
         void igSetScrollHereX(float center_x_ratio = 0.5f);
         bool igSliderScalarN(const(char)* label, ImGuiDataType data_type, void* p_data, int components, const void* p_min, const void* p_max, const(char)* format = null, ImGuiSliderFlags flags = ImGuiSliderFlags.None);
         void ImDrawList_PathBezierQuadraticCurveTo(ImDrawList* self, const ImVec2 p2, const ImVec2 p3, int num_segments = 0);
-        const ImWchar* ImFontAtlas_GetGlyphRangesChineseSimplifiedCommon(ImFontAtlas* self);
+        ImWchar* ImFontAtlas_GetGlyphRangesChineseSimplifiedCommon(ImFontAtlas* self);
         void igGetMousePosOnOpeningCurrentPopup(ImVec2* pOut);
         bool igVSliderScalar(const(char)* label, const ImVec2 size, ImGuiDataType data_type, void* p_data, const void* p_min, const void* p_max, const(char)* format = null, ImGuiSliderFlags flags = ImGuiSliderFlags.None);
         void igDockBuilderSetNodePos(ImGuiID node_id, ImVec2 pos);
@@ -3589,7 +3589,7 @@ version (BindImGui_Static) {
         char* igImStrdup(const(char)* str);
         int igImFormatStringV(char* buf, size_t buf_size, const(char)* fmt, va_list args);
         void igSetTooltipV(const(char)* fmt, va_list args);
-        const ImGuiDataTypeInfo* igDataTypeGetInfo(ImGuiDataType data_type);
+        ImGuiDataTypeInfo* igDataTypeGetInfo(ImGuiDataType data_type);
         bool igVSliderInt(const(char)* label, const ImVec2 size, int* v, int v_min, int v_max, const(char)* format = "%d", ImGuiSliderFlags flags = ImGuiSliderFlags.None);
         void igSetWindowClipRectBeforeSetChannel(ImGuiWindow* window, const ImRect clip_rect);
         ImFontGlyphRangesBuilder* ImFontGlyphRangesBuilder_ImFontGlyphRangesBuilder();
@@ -3670,7 +3670,7 @@ version (BindImGui_Static) {
         ImGuiTableSettings* igTableSettingsFindByID(ImGuiID id);
         void ImGuiIO_AddInputCharacterUTF16(ImGuiIO* self, ImWchar16 c);
         float* ImGuiStorage_GetFloatRef(ImGuiStorage* self, ImGuiID key, float default_val = 0.0f);
-        const ImWchar* igImStrbolW(const ImWchar* buf_mid_line, const ImWchar* buf_begin);
+        ImWchar* igImStrbolW(const ImWchar* buf_mid_line, const ImWchar* buf_begin);
         ImGuiStackSizes* ImGuiStackSizes_ImGuiStackSizes();
         bool igSliderBehavior(const ImRect bb, ImGuiID id, ImGuiDataType data_type, void* p_v, const void* p_min, const void* p_max, const(char)* format, ImGuiSliderFlags flags, ImRect* out_grab_bb);
         void igValue_Bool(const(char)* prefix, bool b);
@@ -3774,7 +3774,7 @@ version (BindImGui_Static) {
         ImVec2* ImVec2_ImVec2_Nil();
         ImVec2* ImVec2_ImVec2_Float(float _x, float _y);
         int ImGuiTextBuffer_size(ImGuiTextBuffer* self);
-        const ImWchar* ImFontAtlas_GetGlyphRangesDefault(ImFontAtlas* self);
+        ImWchar* ImFontAtlas_GetGlyphRangesDefault(ImFontAtlas* self);
         void igUpdatePlatformWindows();
         void igTextWrapped(const(char)* fmt, ...);
         void ImFontAtlas_ClearTexData(ImFontAtlas* self);
